webhook start failed

# Expected Behavior
webhook worker normally

# Actual Behavior
webhook Error
log:
```
I0301 05:29:20.172954       1 server.go:41] 
-------------------------------------------------------------------------------
Kube-OVN: 
  Version:       v1.10.0
  Build:         2022-03-01_13:26:06
  Commit:        git-024d168
  Go Version:    go1.17.7
  Arch:          amd64
-------------------------------------------------------------------------------
/kube-ovn/kube-ovn-webhook flag redefined: log_dir
panic: /kube-ovn/kube-ovn-webhook flag redefined: log_dir

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc00007ec00, {0x56501f5ae970, 0x56502070a0b0}, {0x56501e02529a, 0x7}, {0x56501e079f7c, 0x2f})
	/usr/local/go/src/flag/flag.go:879 +0x2f4
flag.(*FlagSet).StringVar(...)
	/usr/local/go/src/flag/flag.go:762
k8s.io/klog/v2.InitFlags(0xc00007ec00)
	/root/go/pkg/mod/k8s.io/klog/v2@v2.30.0/klog.go:429 +0x55
github.com/kubeovn/kube-ovn/cmd/webhook.CmdMain()
	/root/kube-ovn/cmd/webhook/server.go:44 +0x1e5
main.main()
	/root/kube-ovn/cmd/cmdmain.go:45 +0x17f
```

# Steps to Reproduce the Problem

1. make build && docker build
2. webhook start with kube-ovn:dev


# Additional Info

- kube-ovn version:
master after commit 024d1684b7e2ce9328621b516c56e14755930f3d


delete klog.InitFlags(nil) ?

--- a/cmd/webhook/server.go
+++ b/cmd/webhook/server.go
@@ -14,6 +14,7 @@ import (
 	ovnv1 "github.com/kubeovn/kube-ovn/pkg/apis/kubeovn/v1"
 	ovnwebhook "github.com/kubeovn/kube-ovn/pkg/webhook"
 	"github.com/kubeovn/kube-ovn/versions"
+	"github.com/spf13/pflag"
 )
 
 const (
@@ -40,9 +41,25 @@ func CmdMain() {
 	var port int
 	klog.Infof(versions.String())
 
-	flag.IntVar(&port, "port", 8443, "The port webhook listen on.")
-	klog.InitFlags(nil)
-	flag.Parse()
+	port = *pflag.Int("port", 8443, "The port webhook listen on.")
+
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
+	klog.InitFlags(klogFlags)
+
+	// Sync the glog and klog flags.
+	pflag.CommandLine.VisitAll(func(f1 *pflag.Flag) {
+		f2 := klogFlags.Lookup(f1.Name)
+		if f2 != nil {
+			value := f1.Value.String()
+			if err := f2.Value.Set(value); err != nil {
+				klog.Fatalf("failed to set flag, %v", err)
+			}
+		}
+	})
+
+	pflag.CommandLine.AddGoFlagSet(klogFlags)
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+	pflag.Parse()
 
 	// set logger for controller-runtime framework
 	ctrl.SetLogger(klogr.New())

--- a/yamls/webhook.yaml
+++ b/yamls/webhook.yaml
@@ -33,7 +33,7 @@ spec:
       hostNetwork: true
       containers:
         - name: kube-ovn-webhook
-          image: "kubeovn/kube-ovn:v1.9.0"
+          image: "kubeovn/kube-ovn:v1.10.0"
           imagePullPolicy: IfNotPresent
           command:
             - /kube-ovn/kube-ovn-webhook

